DESCRIPTION.md

```diff
@@ -11,6 +11,7 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 - v3.0.5(TBD)
   - Added the ability to read Snowflake's central configuration file.
   - Improved OCSP response caching to remove tmp cache files on Windows.
+  - Added a parameter ``server_session_keep_alive`` in ``Connection`` that skips session deletion when client connection closes.
```

Comment:
```suggestion
  - Added a parameter ``server_session_keep_alive`` in ``SnowflakeConnection`` that skips session deletion when client connection closes.
```

Reply:
Interesting. Let me change it to single back-ticks to be consistent then :)